internal/controller/streams.go

```diff
@@ -394,6 +395,7 @@ func (sm *streamManager) findHubbleRelay(ctx context.Context, logger *zap.Logger
 	// TODO: Add logic for a discoveribility function to decide which CNI to use.
 	ciliumFlowCollector, err := newCiliumFlowCollector(ctx, logger, ciliumNamespace)
 	if err != nil {
+		logger.Error("Failed to create CiliumFlowCollector", zap.Error(err))
```

Comment:
Logs at info level and above are meant to be readable by end users. Don't use Golang symbols in those.
```suggestion
		logger.Error("Failed to create Cilium flow collector", zap.Error(err))
```